src/components/avatar.tsx

```diff
@@ -0,0 +1,18 @@
+import { twJoin } from "tailwind-merge";
+
+interface AvatarProps {
+  imageUrl?: string;
+  className?: string;
+}
+
+export function Avatar(props: AvatarProps) {
+  const { imageUrl, className } = props;
+
+  return (
+    <img
+      src={imageUrl}
+      alt="userPhoto"
+      className={twJoin("avatar", className)}
```

Comment:
I ran the project and the avatar is not rendering a rounded image, but instead, it's a square. Please, fix the base default classes to make the image to be a circle.

Here's a screenshot of the current state
![image](https://github.com/cristianoprogramador/FE_SH/assets/49873623/10a5bd81-9934-4f54-9e14-1e53a652a7d3)

Reply:
i just fixed that, thanks